winter/Code/palace/palace.cpp

```diff
@@ -234,7 +249,7 @@ int palace::launch_game(const std::string& game_name, const std::string& argumen
       A_error("%s proton version couldn't be found\n", proton_app_name);
       return 1;
     }
-    setenv("STEAM_COMPAT_DATA_PATH", (sys::GetSteamPath() / "steamapps" / "compatdata").string().c_str(), 1);
+    setenv("STEAM_COMPAT_DATA_PATH", (steamPath/ "steamapps" / "compatdata").string().c_str(), 1);
     setenv("STEAM_COMPAT_CLIENT_INSTALL_PATH", sys::GetSteamPath().string().c_str(), 1);
```

Comment:
Missed an instance of sys::GetSteamPath

Reply:
fixed in [d302636](https://github.com/AdastralGroup/osprey/pull/23/commits/d302636707110f0a3303075bd39a39aa75c0fc2f)